src/overlays/actors/ovl_Obj_Kzsaku/z_obj_kzsaku.c

```diff
@@ -28,27 +34,99 @@ const ActorInit Obj_Kzsaku_InitVars = {
     (ActorFunc)ObjKzsaku_Draw,
 };
 
-#endif
-
-extern UNK_TYPE D_06000040;
-extern UNK_TYPE D_06001118;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/ObjKzsaku_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/ObjKzsaku_Destroy.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08B60.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08B7C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08BBC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08BD0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08C84.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08CB0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/ObjKzsaku_Update.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/ObjKzsaku_Draw.s")
+extern Gfx D_06000040[];
+extern CollisionHeader D_06001118;
+
+void ObjKzsaku_Init(Actor* thisx, GlobalContext* globalCtx) {
+    ObjKzsaku* this = THIS;
+    s32 pad;
```

Comment:
```suggestion
    s32 pad;
    ObjKzsaku* this = THIS;
```
Does this match? Usually it's this way when required.